conf/test_pna.config

```
@@ -16,6 +16,11 @@ process {
         memory: '15.GB',
         time: '1.h',
     ]
+
+    // Run a smaller experiment summary for testing
+    withName: EXPERIMENT_SUMMARY {
+        ext.args = "-P test_mode=\"TRUE\""
```

```suggestion
        ext.args = '-P test_mode="TRUE"'
```
Nitpick but avoiding escaping is always nice